### PR TITLE
Make for authenticated websocket

### PR DESCRIPTION
**Pull request by Luckvery**
Just adding an entry for running the authenticated websocket code in the demo project.

### CODE REVIEW

File: cli/src/main/kotlin/com/westwinglabs/coinbase/cli/Launcher.kt [hunk 1]
```diff
@@ -15,12 +15,15 @@ const val OPTION_SAMPLE_SIGNATURE = "sample_signature"
 const val OPTION_SIGNATURE_METHOD = "method"
 const val OPTION_SIGNATURE_PATH = "path"
 const val OPTION_SIGNATURE_BODY = "body"
+const val OPTION_WEBSOCKRT_CHANNELS = "channels"
```
Comment (zugaldia) [inline cli/src/main/kotlin/com/westwinglabs/coinbase/cli/Launcher.kt:18]: Typo on "websocket."

**Review by zugaldia — CHANGES_REQUESTED** [review]
Good example @Luckvery! There's a small typo, once it gets fixed (let me know if you want me to do it) I can get this merged. Being an example it won't necessarily need a new release.

File: cli/src/main/kotlin/com/westwinglabs/coinbase/cli/Launcher.kt (hunk 1 above)
Comment (Luckvery) [reply cli/src/main/kotlin/com/westwinglabs/coinbase/cli/Launcher.kt:18]: Whoops!! 🤦

Comment (Luckvery) [reply cli/src/main/kotlin/com/westwinglabs/coinbase/cli/Launcher.kt]: Fixed!!

**Review by zugaldia — APPROVED** [review]

**Comment by zugaldia** [comment]
:rocket: merged @Luckvery !

**Pull request merged**